packages/react/src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import 'hds-core';
+import styles from './Breadcrumb.module.scss';
+import { Link } from '../link';
+import { IconAngleLeft, IconAngleRight } from '../../icons';
+import classNames from '../../utils/classNames';
+
+export type BreadcrumbListItem = { title: string; path: string | null };
+export type BreadcrumbProps = { list: BreadcrumbListItem[]; label?: string };
```

Comment:
Maybe `ariaLabel` is more descriptive of what it's for. Also JSDocs with short descriptions for these props would serve the users nicely.

Reply:
Changed.